pkg/helpers/docker/docker.go

```diff
@@ -196,18 +205,26 @@ func DownloadAndExtractDockerImage(ctx luettypes.Context, image, dest string, au
 }
 
 func ExtractDockerImage(ctx luettypes.Context, local, dest string) (*images.Image, error) {
+	var img v1.Image
 	if !fileHelper.Exists(dest) {
 		if err := os.MkdirAll(dest, os.ModePerm); err != nil {
 			return nil, errors.Wrapf(err, "cannot create destination directory")
 		}
 	}
 
-	ref, err := name.ParseReference(local)
-	if err != nil {
-		return nil, err
+	var err error
+	if strings.HasPrefix(local, filePrefix) {
```

Comment:
This is looking good! I think it's safe to just check the input for the file suffix. No need to specify --local at that point

Reply:
updated.

_DownloadAndExtractDockerImage_ will be called only if it's `!local && !strings.HasPrefix(image, filePrefix)`
rest of the cases - _ExtractDockerImage_ will be called.